ts/components/dialog/OnionStatusPathDialog.tsx

```diff
@@ -69,7 +69,7 @@ const OnionPathModalInner = () => {
             {nodes.map((snode: Snode | any, index: number) => {
               let labelText = snode.label
                 ? snode.label
-                : countryLookup.byIso(ip2country(snode.ip))?.country;
+                : countryLookup.byIso(ip2country(snode.ip))?.country + ' [' + snode.ip + ']';
```

Comment:
can you change that line to that so the lint is fine?

```ts
: `${countryLookup.byIso(ip2country(snode.ip))?.country} [${snode.ip}]`;
```

Reply:
> can you change that line to that so the lint is fine?
> 
> ```ts
> : `${countryLookup.byIso(ip2country(snode.ip))?.country} [${snode.ip}]`;
> ```

Sorry about that. I didn't have `tslint` installed; and I am, in any case, not a TypeScript or JavaScript programmer, as will become painfully obvious if I continue to contribute. ;-)

The PR has been rebased.

Cheers.